tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -370,7 +370,9 @@ After Docker is installed, launch a Docker container with the TensorFlow binary
 image as follows.
 
 ```bash
-$ docker run -it -p 8888:8888 gcr.io/tensorflow/tensorflow
+$ docker run -d -p 8888:8888 --name="tensor" gcr.io/tensorflow/tensorflow
+
+$ docker exec -it tensor bash
```

Comment:
So I think there are two questions here, if I'm reading correctly:

1. If we start the notebook as a server, should we use `-it` or `-d`? I agree that `-d` is nice, the only downside is that it means it's harder to find the logs. Maybe I'm more interested in reading the logs than other folks, so that's probably fine. 😉 

1. If we do want to start a CLI, how should we do it? I think there are actually *two* use cases: one is "I **just** want a bash prompt", and the other is "I want to get a bash prompt to my **already running** container". For the former, 

  ```
$ docker run -it -p 8888:8888 --entrypoint /bin/bash gcr.io/tensorflow/tensorflow
```
  seems nice; for the latter, I think we want to `docker exec --name=tensor bash`, right?